server/src/lsp-server.ts

```diff
@@ -422,15 +422,24 @@ export class LspServer {
             throw new Error("The document should be opened for completion, file: " + file);
         }
 
-        const result = await this.interuptDiagnostics(() => this.tspClient.request(CommandTypes.Completions, {
-            file,
-            line: params.position.line + 1,
-            offset: params.position.character + 1,
-            includeExternalModuleExports: true,
-            includeInsertTextCompletions: true
-        }));
-        const body = result.body || [];
-        return body.map(entry => asCompletionItem(entry, file, params.position, document));
+        try {
+            const result = await this.interuptDiagnostics(() => this.tspClient.request(CommandTypes.Completions, {
+                file,
+                line: params.position.line + 1,
+                offset: params.position.character + 1,
+                includeExternalModuleExports: true,
+                includeInsertTextCompletions: true
+            }));
+            const body = result.body || [];
+            return body.map(entry => asCompletionItem(entry, file, params.position, document));
+        } catch (error) {
+            if (error.message === "No content available.") {
+                this.logger.warn('No content was available for completion request');
```

Comment:
Do we need to log? Clients will see them in the output panel, they are more noise than helpful.

Reply:
Well, could be useful for debugging. Do you prefer we make it `this.logger.info` ?